lib/matplotlib/dates.py

```diff
@@ -1314,11 +1332,34 @@ def get_locator(self, dmin, dmax):
         #  bysecond, unused (for microseconds)]
         byranges = [None, 1, 1, 0, 0, 0, None]
 
+        # Required attributes to get width from figure's normal points
+        axis_name = getattr(self.axis, 'axis_name', '')
+        axes = getattr(self.axis, 'axes', None)
+        label = getattr(self.axis, 'label', None)
+        figure = getattr(self.axis, 'figure', None)
+        transfig = getattr(figure, 'transFigure', None)
+        maxwid = rcParams['figure.figsize'][0]
+
+        # estimated font ratio on font size 10
+        if label is not None:
+            font_ratio = label.get_fontsize() / 10
+        else:
+            font_ratio = rcParams['font.size'] / 10
```

Comment:
I'm pretty sure the ticks have a fontsize associated with them beyond the label fontsize.  THats the fontsize you should querry, and I'm pretty sure it exists.